pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2426201042>#bb8fd9</a><span> - 10/21/2024, 03:34 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The hardcoded configuration path '/root' may not be suitable for all environments and could pose security risks.
</div>
<div id="fix">
<b>Fix:</b> Implement a more flexible configuration path strategy that works across different environments. Use environment variables or command-line flags to specify the config path.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -28,7 +28,10 @@
  		viper.SetConfigName("config")
  		viper.SetConfigType("yaml")
 -		viper.AddConfigPath("/root")
 +		viper.AddConfigPath("/root") // Docker path
 +		viper.AddConfigPath(".")     // Current directory
 +		viper.AddConfigPath("$HOME/.productivite") // Home directory
 +		viper.AddConfigPath("/etc/productivite")   // System-wide directory
 
  		if err := viper.ReadInConfig(); err != nil {
  			panic(err)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged